config_defaults_inc.php

```diff
@@ -851,13 +851,27 @@
 $g_window_title = 'MantisBT';
 
 /**
+ * OpenSearch engine title prefix.
+ * This is used to describe Browser Search entries, and must be short enough
+ * so that when inserted into the 'opensearch_XXX_short' language string, the
+ * resulting text is 16 characters or less, to be compliant with the limit for
+ * the ShortName element as defined in the OpenSearch specification.
+ * @link http://www.opensearch.org/Specifications/OpenSearch/1.1
+ * @see $g_window_title
+ * @global string $g_search_title
+ */
+$g_search_title = '%window_title%';
```

Comment:
> I don't see that much value in it

It makes configuration easier for admins, by avoiding the need to duplicate declaration of several related options.  Consider the case of paths, or `$g_cookie_prefix` for example. Without the '%%' feature, the admin would have to manually set each of the cookies names in their config_inc.php.

You will probably and rightly argue that we do not need to make the cookies names configurable. Of course by refactoring the code (in the above case, getting rid of these config options), we can implement alternative that works without the recursion.

The questions are, whether this is worth the effort and what kind of performance improvement we can obtain by removing this feature.

> expect a better performance if config_eval would be removed and config_get_global and config_get would be simplified.

If this is important to you, I would suggest to open an issue in the tracker, and evaluate how much we'd gain with your proposed approach.